docs/contributing/adding-images-to-the-wesite.md

```diff
@@ -0,0 +1,40 @@
+# Adding images to the design system website
+When deciding whether to add or amend guidance on the design system website wherein you want to include an image in your change, it's important to consider the impact of using images. They can have implications for web performance and accessibility if used improperly.
+
+Before reading this guidance you should also consult our [image guidance](https://design-system.service.gov.uk/styles/images/) for details on when it's appropriate to use images and how best to use them.
+
+## Decide if your content needs images
+We always try to avoid adding new images to our content. You should only add images if they are the only way to convey information.
+
+In styles, components and patterns, use code examples to convey what our styles, components and patterns will look like. Avoid using screenshots of styles, components and patterns wherever possible.
+
+## Use an SVG file if possible
+If your image is a "graphic" (in other words it isn't a photo or a screenshot), an SVG file will likely be the best format for adding it to the design system website. It'll allow it to scale without loss of detail across different screen sizes and is less likely to impact page performance.
```

Comment:
```suggestion
Use an SVG file when you’re adding a simple graphic or table to the design system website (rather than a photo or screenshot). An SVG file is usually the best format as it allows the image to scale without loss of detail across different screen sizes and is less likely to impact page performance.
```